src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -245,10 +222,32 @@ export default function formReducer(state = initialState, action) {
           [action.pageKey]: 'home',
         };
       }
+      let newPages = state.pages;
```

Comment:
I moved this from the other reducer because with the hook, we aren't using that form data update action, but everything is using this action still.